tk_toolchain/repo.py

```diff
@@ -123,7 +123,7 @@ def is_config(self):
 
         :returns: ``True`` is the repository is for a configuration, ``False`` otherwise.
         """
-        return self.name.startswith("tk-config")
+        return self._folder_contains("core") and self._folder_contains("env")
```

Comment:
Azure pipeline does not name the folder into which your code is cloned after the repo, so shouldn't use this euristic.